scripts/particle_rugosity/particle_rugosity.py

```diff
@@ -0,0 +1,68 @@
+# script for calculating the particle rugosity for a given crystal structure (.cif in local working directory, or refcode of a CSD entry)
+# requires a CCDC license, and loading of the CSD-python miniconda environment if run from Linux command line
+
+# import ccdc modules
+import ccdc
+from ccdc.io import CrystalReader
+from ccdc import particle
+from ccdc.particle import Surface
+from ccdc import morphology
+from ccdc.morphology import BFDHMorphology
+
+def particle_rugosity(crystal):
+# load structure, either (1) crystal structure file in working directory, or 
+# (2) CSD refcode
+    head, sep, tail = crystal.partition('.')
+    if sep == '.':
+    # agrument is crystal structure file 
+        xname = crystal
+        xtal1 = CrystalReader(xname)
+        xtal = xtal1[0]
+        xtal1.close
+    else:
+    # agrument is a CSD refcode 
+        xname = crystal
+        xtal = CrystalReader('CSD').crystal(xname)
+
+    print(f"The crystal structure {crystal} has been loaded")
+    print("Calculating particle rugosity ...")
+# run BFDH morphology to determine predicted particle surface area = hkl planes, and d-spacing of the planes
+    morphology = BFDHMorphology(xtal)
+    facets = morphology.facets
+    f = facets[0]
```

Comment:
Is this still needed?

Reply:
I'm not sure I understand what part of this could be removed

Reply:
oh sorry, I didn't mean to highlight so many lines there. `f = facet[0]`
I don't believe it's doing anything there 🤔